deploy/namespace.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ .Namespace }}
+  name: {{ .HandlerNamespace }}
```

Comment:
I think we have to put the prefix to the namespace too, if not the other operator will remove both namespaces, but correct way to do it is override at Makefile `HANDLER_NAMESPACE` variable.

Reply:
I struggled with this one. I was thinking it should be there, but to me, the namespace is the one place where one can pass the whole thing, at once.  But I'm fine either way.